packages/caip/src/assetId/assetId.ts

```diff
@@ -160,5 +160,12 @@ export const fromAssetId: FromAssetId = assetId => {
   }
 }
 
+export const deserializeNftAssetReference = (
+  assetReference: string,
+): [address: string, id: string] => {
+  const [address, id] = assetReference.split('/')
+  return [address, id]
+}
```

Comment:
To be consumed for the https://github.com/shapeshift/web/pull/4555 diff if it goes in first